### DISCUSSION TITLE
Why does Open Shell take so long to install???

### DISCUSSION THREAD

**Discussion by edycrp** [Open-Shell/Open-Shell-Menu#2058] [General]
Greetings. I have the following question: From Classic Shell until they changed to Open Shell I have used the program and I love it. I install it on my PC and on my clients, but I don't set them to start OpSh from the beginning, since I only use certain functions when I use it with them and I usually leave it off, but on my PC the program starts from the beginning. I use it with another great program which is the explorer patcher, but I don't install EPatcher on the clients anymore because of the black screen program that it gives if it is not updated in time and since the clients don't know what that program is, they don't update it and that's why I don't use it anymore, only on my personal PCs.
I love the function in which OpSh looks like the Windows XP style start menu, because I think it's the most practical start menu of all those that have come out.
The question I have is when I have to install it, because it doesn't matter how fast the computer is, it always takes a long time to install, obviously depending on the speed of the PC on which I install it it is faster or slower and that is why I ask why it takes so long. I have installed it on different PCs and laptops of different brands and models and speeds and memories and hard drives and architectures and versions of Windows, going from Win7, 8, 10, 11 and it always takes a long time, it doesn't matter if it is a clean installation or on top of the previous one.
Could you give me some information to know how to install it faster?
PS: Right now I am installing it and I started installing it 15 minutes before writing this text and I am still writing and it has not finished installing.
Greetings and thanks.

**Comment by ge0rdi** [comment]
> Right now I am installing it and I started installing it 15 minutes before writing this text and I am still writing and it has not finished installing.

That's definitely very long time.
For comparison, I just started Windows Sandbox and installed latest `Open-Shell` in like 15 seconds.

I guess, there is something on your systems that somehow interact with the installation.

You can try to create log from installation by adding `/lvx* install.log` to installer's command line.
Maybe it can shed some light on what takes so long on your machine.
